eduNEXT/tutor-contrib-ontask
# refactor!: move settings to apps sub-directories

### Description
This PR moves the current settings to their own directory. It also adds a volume called production.py, allowing us to override production environment settings.

### How to test
Generate your environment dev|local, it should work the same. For the settings override, add to your config.yml:
```
ONTASK_EXTRA_PRODUCTION_SETTINGS = |
"""
LOGGING = # Login override...
"""
```
